### PR TITLE
Added test for container entity

### PR DESCRIPTION
**Pull request by schlupov**

### CODE REVIEW

**Comment by pondrejk** [comment]
first error line from travis addressed elsewhere https://github.com/SatelliteQE/robottelo/pull/6237

File: tests/foreman/ui_airgun/test_container.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+)
+
+
+@fixture(scope='module')
+def module_org():
+    return entities.Organization().create()
+
+
+@fixture(scope='module')
+def create_docker(module_org):
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_container.py]: `module_docker_cr`?

File: tests/foreman/ui_airgun/test_container.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    docker_url = settings.docker.external_url
+    return entities.DockerComputeResource(
+        name=gen_string('alpha'),
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_container.py]: redundant line, name will be generated automatically if not specified

File: tests/foreman/ui_airgun/test_container.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    :id: 299fc13d-5c2f-4642-a6de-735c89fdd096
+
+    :expectedresults: The docker container is created for the compute
+        resource
+
+    :CaseLevel: Integration
+    """
+    deploy_on = create_docker.name + ' (Docker)'
+    lifecycle_environment = ENVIRONMENT
+    content_view = DEFAULT_CV
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_container.py]: Pls avoid creating single-use variables and just use those constants directly.
Moving variables to the top of test makes sense when there's a chance you'll change those values. But that's not the case here - as your test doesn't create lce or cv you can't change these vars without updating rest of the test (unlike, let's say, `upstream_repository_name`).

File: tests/foreman/ui_airgun/test_container.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        name=repo_name,
+        url=DOCKER_REGISTRY_HUB,
+        product=module_prod,
+        content_type=REPO_TYPE['docker'],
+        docker_upstream_name=upstream_repository_name
+    ).create()
+    repo.sync()
+    with session:
+        session.container.create({
+            'ContainerPreliminaryCreateView.compute_resource.deploy_on': deploy_on,
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_container.py]: don't forget to update view names after addressing comments in SatelliteQE/airgun#199

**Comment by schlupov** [comment]
screenshot from 2018-09-24 14-35-51
Test passed, a container was successfully created.

**Comment by abalakh** [comment]
@schlupov please rebase your branch to fix the travis

File: tests/foreman/ui_airgun/test_container.py [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+            'preliminary.compute_resource.deploy_on': deploy_on,
+            'image.content_view.lifecycle_environment': ENVIRONMENT,
+            'image.content_view.content_view': DEFAULT_CV,
+            'image.content_view.repository': repo.name,
+            'image.content_view.tag': 'centos7.2.1511',
+            'image.content_view.capsule': settings.server.hostname,
+            'configuration.name': gen_string('alpha'),
+            'configuration.command': 'top',
+            'environment.tty': True,
+        })
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_container.py]: your test doesn't contain any assertion. Pls ensure your container can be found using search (at least) or add support of `read()`, read it's values and make sure they're the same as during creation (preferably).

**Comment by schlupov** [comment]
tests/foreman/ui_airgun/test_container.py::test_positive_create_with_compresource PASSED                                                                                               [100%]

@abalakh Fixed assert. Please take a look. Now it works.

**Pull request merged**